### DISCUSSION TITLE
Standardize list endpoint response

### DISCUSSION THREAD

**Discussion by rishighan** [bpepple/metron#81] [API Suggestions]
I wanted to start a conversation about standardizing the response for endpoints to have consistent keys.
Specifically referencing the `/api/publisher` endpoint (https://github.com/bpepple/metron/issues/80#issuecomment-1005383218)

**Comment by bpepple** [comment]
Copying over my response from the issue for the discussion:

The `__str__` method is used for the display name for an object, and is not the same as the db field `name`. In your example above, the `name` field would be `Wolverine: Black, White & Blood`. The reason I used the object display name is to easily differentiate multiple results (for example if someone where to do a series query for `Batman`).

To fix this I would probably need to add the `name`, `year`, and `volume` field to the serializer (and possibly removing or renaming `__str__` to something like `display_name`)

**Comment by bpepple** [comment]
This suggestion should probably also address other resources that also use an objects display name, which off the top of my head is only the `Issue` resource (tho I'll need to verify that). In terms of implementing it, this **will** require change the existing API,  so there could be unforeseen complications and I'll also need to modify any relevant tooling using those endpoints.

**Reply by rishighan** [reply]
That's quite alright, I am mitigating it trivially in my app, so we can keep this discussion open.
